### ISSUE TITLE
[Bug?]: eslint config

### ISSUE THREAD

**Issue by ejsinfuego** [redwoodjs/redwood#11763]
### What's not working?

I already made it work through reading this [eslint-config](https://github.com/redwoodjs/redwood/blob/v8.4.1/packages/eslint-config/README.md#overriding-default-configuration). I don't know if this only happens on my end but I would have to add 
```web/config/.eslintrc.js``` with contents:

```
module.exports = {
  extends: '../../.eslintrc.js',
}
```
to make it work. 

### What's your environment? (If it applies)

```shell
System:
    OS: Windows 10 10.0.19045
  Binaries:
    Node: 20.11.0 - ~\AppData\Local\Temp\xfs-001d3afe\node.CMD
    Yarn: 4.1.1 - ~\AppData\Local\Temp\xfs-001d3afe\yarn.CMD
  Browsers:
    Edge: Chromium (127.0.2651.74)
  npmPackages:
    @redwoodjs/auth-dbauth-setup: 7.6.2 => 7.6.2
    @redwoodjs/cli-storybook: 7.6.2 => 7.6.2
    @redwoodjs/core: 7.6.2 => 7.6.2
```

### Are you interested in working on this?

- [X] I'm interested in working on this

**Comment by Tobbe** [comment]
> to make it work.

To make what work? 😅  I feel really stupid here, but I don't quite understand what you're trying to do

**Comment by ejsinfuego** [comment]
I'm sorry for the confusion. This is a problem with eslint config. I can submit a pr if that would help? 😅

**Comment by Tobbe** [comment]
Yes please. A PR would be great :) Having some code to look at always helps
